odoo/addons/base/__manifest__.py

```diff
@@ -84,6 +84,7 @@
         'data/res_partner_demo.xml',
         'data/res_partner_image_demo.xml',
     ],
+    'depends': [],
```

Comment:
I treat `'depends': []` and no `depends` modules as `'depends': ['base'],`
and `base` module is the only exception in which the hack `'depends': []` is allowed
otherwise {'base': {'depends': ['base']}} is a depends loop

Reply:
What I meant is: I don't see the point of making it explicit.